docs/content/developer/getting-started/independent-ticketing-system/frontend.mdx

```diff
@@ -0,0 +1,220 @@
+---
+title: Frontend
+description: Tutorial for Independent Ticketing System frontend 
+---
+
+# Independent Ticketing System Frontend
+
+Till now you have build the package contract of Independent ticketing system, its time to create the UI. To create the frontend DApp we use [IOTA dApp kit](../../../ts-sdk/dapp-kit/index.mdx), apart from dApp kit we will also use an external package for client side routing which is react-router-dom.
+
+### Setup the Frontend dApp
+
+First create an initial react app using the following command
+
+```npm2yarn
+npm create vite@latest
+```
+
+Run the following command to install the [IOTA dApp kit](../../../ts-sdk/dapp-kit), [IOTA SDK](../../../ts-sdk/typescript), and [@tanstack/react-query](https://www.npmjs.com/package/@tanstack/react-query) in the react app.
+
+```npm2yarn
+npm add @iota/dapp-kit @iota/iota-sdk @tanstack/react-query
+```
+
+### Setting Up Network Configuration
+
+Add the following variables in the [`networkConfig.ts`](https://github.com/iota-community/independent-ticketing-system/blob/main/independent_ticketing_system/Frontend/src/networkConfig.ts) file:
+
+```javascript reference
+https://github.com/iota-community/independent-ticketing-system/blob/4eabf7ef0c74e843d95e1658ae071df34488fd04/independent_ticketing_system/Frontend/src/networkConfig.ts#L1-L27
+```
+
+* All the shared objects mentioned above were generated during the package publishing process.
```

Comment:
```suggestion
You should retrieve the shared objects from the transaction digest of the published [package](package.mdx).
```
You could also add a note asking the users to copy these from the digest when you tell them to publish

Please add a link to the particular headder